### PR TITLE
[#1] Initialize the project

### LINKED ISSUE #1: Initialize the project
## Why

Initialize the project so as to support the staging and production releases.

## Acceptance Criteria

- Initialize the Flutter project with [flutter-templates](https://github.com/nimblehq/flutter-templates)

### LINKED ISSUE #3: Configure project based on templates
## Why

To make the project align with the team's convention, configuring the project based on the available templates is needed.

## Acceptance Criteria

Configure the project based on templates:

- [flutter-template](https://github.com/nimblehq/flutter_templates)
- [git-template](https://github.com/nimblehq/git-template)

### PR DESCRIPTION
**Pull request by nmint8m**
- Close #1 

## What happened 👀

Initialize the project so as to support the staging and production releases.

## Insight 📝

- Initialize the Flutter project with [flutter-templates](https://github.com/nimblehq/flutter-templates)

### CODE REVIEW

**Comment by doannimble** [comment]
@nmint8m I think for saving time, we only need to take care of stagging. So next time, please do not spend time making POW of production 🙏

**Review by doannimble — APPROVED** [review]

File: bin/easy_install [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/Users/nmint8m/Repositories/Nimble/flutter_templates/bin/python3
```
Comment (manh-t) [inline bin/easy_install]: we should put `./bin` into `.gitignore`

File: lib/python3.8/site-packages/easy_install.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+"""Run the EasyInstall command"""
```
Comment (manh-t) [inline lib/python3.8/site-packages/easy_install.py]: redundant folder 🙋

File: pyvenv.cfg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+home = /Users/nmint8m/Repositories/Nimble/flutter_templates/.venv/bin
```
Comment (manh-t) [inline pyvenv.cfg]: this file is redundant as well

File: scripts/setup.py [hunk 4]
```diff
@@ -0,0 +1,442 @@
+#!/usr/bin/env python3
```
Comment (manh-t) [inline scripts/setup.py]: after generating the project, we can remove `scripts/` 😄

File: bin/easy_install (hunk 1 above)
Comment (manh-t) [reply bin/easy_install]: @nmint8m sorry for not clear enough 😓 we also need to remove `bin/` folder out of the version control 🙏

File: lib/python3.8/site-packages/easy_install.py (hunk 2 above)
Comment (manh-t) [reply lib/python3.8/site-packages/easy_install.py]: same, we should remove `python3.8` out of git

File: pyvenv.cfg (hunk 3 above)
Comment (manh-t) [reply pyvenv.cfg]: same here

**Review by manh-t — APPROVED** [review]

**Comment by nmint8m** [comment]
Solve #3

**Pull request merged**